src/pages/Graph/GraphPage.tsx

```diff
@@ -115,21 +115,23 @@ export default class GraphPage extends React.Component<GraphPageProps, GraphPage
   }
 
   componentDidUpdate(prevProps: GraphPageProps) {
-    const prevNamespace = prevProps.namespace;
+    // @ts-ignore
+    const prevActiveNamespace = prevProps.activeNamespace;
```

Comment:
You need to add `activeNamespace: Namespace;` to `GraphPageProps` (same file) to remove the ts-ignores from this file.

Reply:
OK, thanks for the tip.